serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/utils/apicurio/DynamicSchema.java

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright 2022 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.amazonaws.services.schemaregistry.utils.apicurio;
+
+import com.google.protobuf.DescriptorProtos;
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.DynamicMessage;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Method;
+import java.util.*;
+
+public class DynamicSchema {
```

Comment:
Is this class really required? Why wasn't this present before?

Reply:
These additional classes in the patch for apicurio are used in the main parser that we need - since I'm not familiar with what the parser does/needs, I just ported over all classes it uses to leave it as untouched as possible. These are classes taken from apicurio, they aren't directly necessary for what we need as far as I'm aware.